tests/SoCreate.ServiceFabric.PubSub.Tests/GivenServiceReference.cs

```diff
@@ -54,5 +54,246 @@ public void WhenDeterminingShouldDeliverMessageToActorWithUnmatchingPayload_Then
             bool shouldDeliver = actorRef.ShouldDeliverMessage(messageWrapper);
             Assert.IsFalse(shouldDeliver);
         }
+        [TestMethod]
+        public void WhenDeterminingShouldDeliverMessageToActorWithUnmatchingPayload_ThenReturnsTrue()
+        {
+            var actorRef = new ActorReferenceWrapper(new ActorReference { ActorId = ActorId.CreateRandom() }, "Customer.Name", "Customer1");
+            var messageWrapper = new
+            {
+                Customer = new
+                {
+                    Name = "Customer1"
+                }
+            }.CreateMessageWrapper();
+
+            bool shouldDeliver = actorRef.ShouldDeliverMessage(messageWrapper);
+            Assert.IsTrue(shouldDeliver);
+        }
+
+        /// Testing Wild Card Support 
+        [TestMethod]
+        public void WhenDeterminingShouldDeliverMessageToServiceWithMatchingPayloadWithWildCard_ThenReturnsTrue()
+        {
+            var serviceRef = new ServiceReferenceWrapper(new ServiceReference(), "Customer.Name", "*");
+            var messageWrapper = new
+            {
+                Customer = new
+                {
+                    Name = "Customer1"
+                }
+            }.CreateMessageWrapper();
+
+            bool shouldDeliver = serviceRef.ShouldDeliverMessage(messageWrapper);
+            Assert.IsTrue(shouldDeliver);
+        }
+        [TestMethod]
+        public void WhenDeterminingShouldDeliverMessageToActorWithUnmatchingPayloadWithWildCard_ThenReturnsTrue()
```

Comment:
Test name should be "WhenDeterminingShouldDeliverMessageToActorWithMatchingPayloadWithWildCard_ThenReturnsTrue".